modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,26 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords: upgrade, promotion, premium, button
+
+== Premium upgrade promotion defaults
+
+When {productname} 6.2 or later is running in self-hosted Community mode, an *Upgrade* promotion button appears in the top-right of the editor menu bar by default. The button does note appear when {productname} is running in Community mode on the Tiny cloud.
+
+This promotion button redirects to a link:{companyurl}/tinymce-self-hosted-premium-features/[promotion page], showcasing the benefits of the variety of xref:plugins#premium-plugins[Premium Plugins] available for {productname}.
+
+When {productname} 6.2 or later is running in Premium mode, the *Upgrade* promotion button is disabled. And it is disabled when running on a Premium plan whether {productname} is running in the {cloudname} or as a self-hosted instance.
+
+To turn this button off in Community mode, set the `promotion` option, which defaults to `true`, to false.
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea",  // change this value according to your HTML
+  promotion: false
+});
+----
+
+The *Upgrade* button appears as part of the {productname} menu bar and, consequently, does not appear if the menu bar is disabled.
+
+IMPORTANT: It is not necessary to set `promotion` to `false` when {productname} is running in Premium mode. The *Upgrade* promotion button is disabled in this instance.
```

Comment:
Would it be clearer with the injection of "automatically" 

> The *Upgrade* promotion button is automatically disabled in this instance

Also, integrator could still turn it on if they wanted to. Do we need the wording to convey that? Or just assume that it's so unlikely that trying to cover that case would just confuse the issue?

Reply:
_Automatically_ was injected as part of the re-write.

Reply:
> integrator could still turn it on if they wanted to.

After more thought and bad re-writes than I like to admit, I’ve added a new section, *Premium upgrade promotion option*.

It includes *Type*, *Default value* and *Possible values* entries and a NOTE explaining that it is possible to set this option contrary to its defaults no matter the distribution being used.